src/platform/utilities/tests/sso.unit.spec.js

```diff
@@ -9,15 +9,20 @@ import * as profUtils from 'platform/user/profile/utilities';
 import { checkAutoSession, checkAndUpdateSSOeSession } from '../sso';
 import * as forceAuth from '../sso/forceAuth';
 
-function setKeepAliveResponse(stub, sessionTimeout = 0) {
+function setKeepAliveResponse(stub, sessionTimeout = 0, csid = null) {
   const response = new Response();
   response.ok = true;
   response.headers.set('content-type', 'application/json');
   response.headers.set('session-alive', 'true');
   response.headers.set('session-timeout', sessionTimeout);
+  response.headers.set('va_eauth_csid', csid);
   response.headers.set(
     'va_eauth_authncontextclassref',
-    sessionTimeout ? 'mhv' : null,
+    {
+      DSLogon: 'NOT_FOUND ',
+      mhv: 'NOT_FOUND ',
```

Comment:
Just a minor detail, but are these supposed to have trailing spaces?

Reply:
I originally put them in there to test that we would trim any extra whitespace.  However the code has been changed to not run that check anymore, so its not needed.